example/src/main/main.py

```diff
@@ -1,9 +1,11 @@
 import sys
+import os
 from bugloc.app import Bar
 
 if __name__ == '__main__':
     print(sys.path)
     print(sys.argv)
+    print(os.getenv('CUSTOM_ENV'))
```

Comment:
Could you also save this Run Configuration as a file to the `example/.run` folder? And add this env var for sure.